### PR TITLE
update for Cordova 5, Android SDK 20+

### LINKED ISSUE #62: Fails to run with latest android SDK
When built with android-sdk 19, the webview creation fails with some null pointer exceptions. Luckily, if you built it with older sdk, it's running successfully running on android 4.4.

### PR DESCRIPTION
**Pull request by alexandervlpl**
I updated the plugin to work with newer versions of Cordova/Android. Tested with this setup:
* ionic 1.6.4
* cordova 5.1.1
* Android SDK 22

This should resolve #62.

### CODE REVIEW

File: platforms/android/src/jp/wizcorp/phonegap/plugin/wizViewManager/WizViewManagerPlugin.java [hunk 1]
```diff
@@ -79,12 +80,14 @@ public void run() {
...
         super.initialize(cordova, webView);
-        webView.post(new Runnable() {
+        webView.getView().post(new Runnable() {
             @Override
             public void run() {
-                webView.getSettings().setDomStorageEnabled(true);
-                webView.getSettings().setLoadWithOverviewMode(true);
-                webView.getSettings().setUseWideViewPort(true);
+		WebSettings settings;
+		settings = ((WebView) webView.getView()).getSettings();
```
Comment (jrouault) [inline platforms/android/src/jp/wizcorp/phonegap/plugin/wizViewManager/WizViewManagerPlugin.java]: Indentation got funky on these two lines

**Comment by jrouault** [comment]
Very cool, thanks mate!
We will test it as soon as you can push the identation fix.

I'm not a big fan of the several calls to ```getView()``` if you can do something about it too, otherwise it should be fine as-is.

**Comment by jrouault** [comment]
I just had a look at ```sendJavascript``` and it looks like it's now deprecated. See here:
https://github.com/apache/cordova-android/blob/4.1.1/framework/src/org/apache/cordova/CordovaWebView.java#L92

Any chance you could update to the preferred way now?

**Comment by alexandervlpl** [comment]
Fixed indentation and minimized calls to `getView()`. Hmm, looks like replacing `sendJavascript()` involves changes on the JS side as well. Maybe you can accept this as an initial, working update and make an issue to remove the deprecated function in the future?

**Comment by jrouault** [comment]
Yeah sure, no worries about this.
Thanks for the changes. I'll test it and merge it asap, ie tomorrow. :+1:

**Comment by jrouault** [comment]
Tested on Cordova 5.2.0 / Cordova Android 4.1.1 and working fine!
Thank you for your contribution!

**Comment by alexandervlpl** [comment]
Glad to help! Great, thanks for the quick merge.

**Pull request merged**